UCLA Radio/UCLA Radio/extensions/DateFormatter+ShowDate.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  DateFormatter+ShowDate.swift
+//  UCLA Radio
+//
+//  Created by Nathan Smith on 12/6/16.
+//  Copyright © 2016 ChrisLaganiere. All rights reserved.
+//
+
+import Foundation
+
+extension DateFormatter {
+    
+    enum DateFormat: String {
+        case DayAndTime = "EEE ha"
+        case Time = "ha"
+    }
+    
+    // Expect string of DayAndTime DateFormat ("EEE ha")
+    func formatShowTimeStringToDateComponents(_ s: String) -> DateComponents? {
```

Comment:
avoid using abbreviations like this (`s`) - Cocoa developers write out the whole variable name: `input`